roles/federation-registry/tasks/deploy.yml

```diff
@@ -33,7 +33,7 @@
       repo: "{{federation_registry_git.git_repo}}"
       version: "{{federation_registry_git.git_branch}}"
       dest: "{{federation_registry_git.clone_dest}}"
-      key_file: "{{ federation_registry_git.key_path }}"
+      accept_hostkey: yes
```

Comment:
Is there a way to avoid `accept_hostkey: yes`?
According to the [ansible documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/git_module.html#parameter-accept_hostkey) : 

> Be aware that this disables a protection against MITM attacks.

I think setting the `repo` to `https://github.com/rciam/rciam-federation-registry.git` would do the trick